core/web/presenters/job.go

```diff
@@ -137,6 +137,7 @@ type OffChainReportingSpec struct {
 	CreatedAt                                 time.Time            `json:"createdAt"`
 	UpdatedAt                                 time.Time            `json:"updatedAt"`
 	EVMChainID                                *utils.Big           `json:"evmChainID"`
+	OCRDatabaseTimeout                        models.Interval      `json:"ocrDatabaseTimeout"`
```

Comment:
Is this an optional value? If so and the default is pulled in from an env var, you will have to add an `OCRDatabaseTimeoutEnv` field here as well to indicate whether it came from an env var

Reply:
Fixed.